packages/orchestrator/internal/sandbox/checks.go

```diff
@@ -2,37 +2,49 @@ package sandbox
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"time"
 
 	"github.com/e2b-dev/infra/packages/shared/pkg/consts"
 	"github.com/e2b-dev/infra/packages/shared/pkg/utils"
+	"golang.org/x/mod/semver"
+)
+
+const (
+	healthCheckInterval      = 10 * time.Second
+	metricsCheckInterval     = 5 * time.Second
+	minEnvdVersionForMetrcis = "0.1.3"
```

Comment:
Sorry, one more "last" thing :D I think the current version of envd in `main` is `0.1.4`. We also updated some fields in here, so I would suggest to increase it to `0.1.5` and also bump it up here